content/blog/building-images-bake/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Creating a custom container image for CloudNativePG v2.0"
+date: 2025-06-17
+draft: false
+image:
+    url: 
+    attribution: 
+author: jgonzalez
+tags:
+ - blog
+ - information
+ - programming
+ - applications
+ - containers
+ - postgresql
+ - postgres
+ - images
+ - tutorial
+ - bake
+ - docker
+summary: Creating a container image for CloudNativePG Operator v2.0
+---
+
+## Summary
+In an almost [two years old blog post]({{% ref "/blog/creating-container-images/" %}}), we explained how
+to build a custom container image for CloudNativePG. After two years, many things have changed in the world of containers.
+One of those things has been the introduction of [Bake](https://docs.docker.com/build/bake/) in Docker, which allows you to build
+images using a simple configuration file. Bake is now our recommended way to build images for CloudNativePG.
+
+We will follow a simple baking recipe to create a custom container image or a set of container images, since Bake
+allows you to build multiple images at once in a simple way.
+
+## Ingredients
+
+- A Bake file. We will use the one provided in the [CloudNativePG repository](https://github.com/cloudnative-pg/postgres-containers/blob/main/docker-bake.hcl)
+- Another Bake file, but this time a local one, this one is to overwrite the previous one.
```

Comment:
To overwrite the previous one? Then why have the other one?
I imagine the second bake file allows you to override specific variables?
Also "but this time" is useless.
You have a base Bake file, and a second Bake file to customize specific settings. No "this time".